### PR TITLE
treewide: Format all Nix code according to RFC-166

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: pills/17/config-nix.txt [hunk 1]
```diff
@@ -1,5 +1 @@
-{
-  packageOverrides = pkgs: {
-    graphviz = pkgs.graphviz.override { withXorg = false; };
-  };
-}
+{ packageOverrides = pkgs: { graphviz = pkgs.graphviz.override { withXorg = false; }; }; }
```
Comment (drupol) [inline pills/17/config-nix.txt:1]: That looks so weird... I don't like so much this change TBH.

**Review by drupol — APPROVED** [review]
LGTM, but I don't like one of the changes.

**Comment by drupol** [comment]
I see you pushed some changes, but is it still compliant with RFC-166 ?

File: pills/17/config-nix.txt (hunk 1 above)
Comment (dasJ) [reply pills/17/config-nix.txt]: Unfortunately, this is what the formatter gives us and I'm not a fan of manually fixing the formatted output.
So I added a comment into the innermost attrset which forces an expansion (which I think is the issue you had)

Comment (drupol) [reply pills/17/config-nix.txt]: That's just perfect now. Thank you !

**Review by drupol — APPROVED** [review]

File: pills/17/config-nix.txt [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   packageOverrides = pkgs: {
-    graphviz = pkgs.graphviz.override { withXorg = false; };
+    graphviz = pkgs.graphviz.override {
+      # enable xorg support
```
Comment (SuperSandro2000) [inline pills/17/config-nix.txt:4]: This should be disable xorg support

**Comment by henrik-ch** [comment]
@dasJ, thank you for the comprehensive update to the repo's formatting—great work, especially given the scope! Apologies for chiming in a bit late on this.

To ensure consistency in future updates and avoid inadvertently diverging from the new standard, could you share the specific command or process you employed for this reformatting? I'm assuming there was some automation involved rather than manual adjustments?

I attempted to replicate the formatting on my end, targeting a pre-update version of the repo, using:

`nix-shell -p nixpkgs-fmt --run 'nixpkgs-fmt .'`

However, this only resulted in changes to a select few files within pill 20, as indicated by my git status output below:

```
❯ git status
HEAD detached at a66fd07
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   default.nix
	modified:   pills/20/three-hellos.nix
	modified:   pills/20/two-hellos.nix
	modified:   release.nix

no changes added to commit (use "git add" and/or "git commit -a")
```

Please could you share the correct approach or any insights into the process you used.

Many thanks in advance! 🙏

**Comment by dasJ** [comment]
Unfortunately, the nix code is not actually in `.nix` files which meant manual work by calling the formatter manually rather than going over the entire tree :/

Also, the soon-to-be default formatter for nixpkgs will be [nixfmt](https://github.com/NixOS/nixfmt/), not nixpkgs-fmt (I know the name is confusing). The new formatter is packaged in nixpkgs right now as `nixfmt-rfc-style` (not to be confused with `nixfmt` which has the "classic" pre-RFC style).

I know it's a mess right now but there is ongoing work to clean this up :)

**Pull request merged**